h1st/model/fuzzy/fuzzy_logic_rules.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from typing import NoReturn
+
+import skfuzzy
+from skfuzzy import control as skctrl
+from skfuzzy.control.term import Term
+
+from h1st.model.fuzzy.enums import FuzzyMembership as fm
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FuzzyLogicRules:
+    def __init__(self) -> None:
+        self.membership = {
+            fm.TRIANGLE: skfuzzy.trimf,
+            fm.TRAPEZOID: skfuzzy.trapmf,
+            fm.GAUSSIAN: skfuzzy.gaussmf,
+        }   
+        self.vars = {}
+        self.rules = {}
+
+    def add_rule(self, name, if_: Term, then_: Term): #  -> NoReturn
```

Comment:
Just `add()` is sufficient. On the other hand, the variable name should be `rule_name`.